sycl/include/CL/sycl/detail/cg.hpp

```diff
@@ -62,7 +163,9 @@ class CG {
     FILL_USM = 11,
     PREFETCH_USM = 12,
     CODEPLAY_INTEROP_TASK = 13,
-    CODEPLAY_HOST_TASK = 14
+    CODEPLAY_HOST_TASK = 14,
+    KERNEL_V1 =
```

Comment:
Probably it makes more sense to not have distinct enum value and hide manipulations with version in setType, getType, setVersion, getVersion methods.